build-clang-with-args.sh

```diff
@@ -69,29 +69,29 @@ cd "${llvm_build_dir}"
 # - Stage 2 Build
 # - Build compiler-rt and other runtimes
 
-llvm_tools="llvm-ar;"
-llvm_tools+="llvm-cov;"
-llvm_tools+="llvm-dwarfdump;"
-llvm_tools+="llvm-nm;"
-llvm_tools+="llvm-objcopy;"
-llvm_tools+="llvm-objdump;"
-llvm_tools+="llvm-profdata;"
-llvm_tools+="llvm-ranlib;"
-llvm_tools+="llvm-readelf;"
-llvm_tools+="llvm-readobj;"
-llvm_tools+="llvm-size;"
-llvm_tools+="llvm-strings;"
-llvm_tools+="llvm-strip;"
-llvm_tools+="llvm-symbolizer"
+llvm_tools="llvm-ar"
+llvm_tools+=";llvm-cov"
+llvm_tools+=";llvm-dwarfdump"
+llvm_tools+=";llvm-nm"
+llvm_tools+=";llvm-objcopy"
+llvm_tools+=";llvm-objdump"
+llvm_tools+=";llvm-profdata"
+llvm_tools+=";llvm-ranlib"
+llvm_tools+=";llvm-readelf"
+llvm_tools+=";llvm-readobj"
+llvm_tools+=";llvm-size"
+llvm_tools+=";llvm-strings"
+llvm_tools+=";llvm-strip"
+llvm_tools+=";llvm-symbolizer"
```

Comment:
I don't think cmake complains about an empty item at the end of the list (at least it didn't complain when I had an extraneous `;` in LLVM_TARGETS_TO_BUILD), so you can just put the `;` at the end of all tools.

Reply:
The idea here is that you ensure the first line is the different one, rather than all the middle ones being the same, and the first and last being slightly special in their own way. Should prevent programmer error, I feel.

Reply:
By programmer I mean "me"